### PR TITLE
Make it beatiful

### PR DESCRIPTION
**Pull request by viniciushisao**
No more to say.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,11 @@ The web server repository can be found [here](https://github.com/marcelorcorrea/
 
 Feel free to fork or contribute this project.
 
-Download in [Play Store](https://play.google.com/store/apps/details?id=org.falaeapp.falae)
+[![Slack channel](https://img.shields.io/badge/Chat-Slack-blue.svg)](https://falaeapp.slack.com)
+[![Language Kotlin](https://img.shields.io/badge/Language-Kotlin-blue.svg)](https://kotlinlang.org/)
+[![License MIT](https://img.shields.io/badge/Licence-Mit-blue.svg)](LICENSE)
+[![Downolad Android](https://img.shields.io/badge/Download-Android-blue.svg)](https://play.google.com/store/apps/details?id=org.falaeapp.falae)
```
Comment (longaraymatheus) [inline README.md]: Nit: `Downolad`

File: README.md [hunk 2]
```diff
@@ -18,7 +18,11 @@ The web server repository can be found [here](https://github.com/marcelorcorrea/
 
 Feel free to fork or contribute this project.
 
-Download in [Play Store](https://play.google.com/store/apps/details?id=org.falaeapp.falae)
+[![Slack channel](https://img.shields.io/badge/Chat-Slack-blue.svg)](https://falaeapp.slack.com)
+[![Language Kotlin](https://img.shields.io/badge/Language-Kotlin-blue.svg)](https://kotlinlang.org/)
+[![License MIT](https://img.shields.io/badge/Licence-Mit-blue.svg)](LICENSE)
```
Comment (longaraymatheus) [inline README.md]: Just for checking, is there a badge link with `License` with 's', and `MIT` in capital letters?

Comment (viniciushisao) [reply README.md]: Yes. There is. This is not static, if you change the name of svg file, you can change it at all.

File: README.md (hunk 1 above)
Comment (viniciushisao) [reply README.md]: yeah, but this is pretty stable, I saw many power git pages using it. Anyway, it will not crash our app if this failed in some time.

**Comment by viniciushisao** [comment]
Vamos integrar, Marcelo?

**Comment by marcelorcorrea** [comment]
Do It.

**Pull request merged**